stdlib/http/src/main/java/org/ballerinalang/net/http/HttpUtil.java

```diff
@@ -490,6 +499,21 @@ public static HttpCarbonMessage createErrorMessage(String payload, int statusCod
         return response;
     }
 
+    public static HttpCarbonMessage createErrorMessage(String payload, int statusCode, String serverName) {
```

Comment:
Seems like this method is only used for tests. Shall we use already existing one. We can do the header setting at the place where which this method is invoked. WDYT?